updatecli/values.yaml

```diff
@@ -0,0 +1,8 @@
+github:
+  user: "updatebot"
+  email: "updatebot@olblak.com"
```

Comment:
Really?

Reply:
Yeah... In our todo list, almost made the remark

Reply:
https://github.com/jenkins-infra/helpdesk/issues/2753 :) I'm sure that @lemeurherve agrees with you \o/